airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/factory/DataSourceFactory.java

```diff
@@ -180,7 +180,8 @@ private static class DataSourceBuilder {
     private String jdbcUrl;
     private int maximumPoolSize = 10;
     private int minimumPoolSize = 0;
-    private long connectionTimeoutMs;
+    // This feels really weird, but it's strictly equivalent to the original long connectionTimeoutMs
```

Comment:
Can we just remove this comment?

Reply:
why ? I do genuinely think having a default set to 0 is weird

Reply:
I think that's worth commenting too but I don't think referencing "the original long connectionTimeoutMs" is going to be useful to the reader.